lib/cloudwatch-logs.js

```diff
@@ -0,0 +1,25 @@
+import AWS from 'aws-sdk'
+
+const cloudwatchLogs = new AWS.CloudWatchLogs({
+  accessKeyId: process.env.AWS_ACCESS_KEY_ID,
+  secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
+  region: process.env.AWS_REGION,
+})
+
+export const cloudwatchPutLogEvents = (params, service = cloudwatchLogs) =>
+  new Promise((resolve, reject) => {
+    service.putLogEvents(params, (error, data) => {
+      if (data) resolve(data)
+      if (error) reject(error)
```

Comment:
Est-ce que l'erreur ne doit pas être traitée avant les `data` ?

Reply:
Dans le documentation de l'API de CloudWatch Logs, il est écrit que l'on reçoit soit l'un, soit l'autre. L'ordre dans lequel on les traite n'est pas vraiment important.

Reply:
Que dis-tu de clarifier ce comportement avec

```suggestion
      else if (error) reject(error)
```